week3/day_10_exercises.R

```diff
@@ -0,0 +1,199 @@
+
+lm.fit <- lm(medv ~ lstat, Boston)
+anova(lm.fit, lm.fit2)
+
+
+par(mfrow = c(2,2))
+plot(lm.fit2)
+
+lm.fit5 <- lm(medv ~ poly(lstat, 5), Boston)
+summary(lm.fit5)
+
+summary(lm(medv ~ log(rm), Boston))
+
+# Lab 3.6.6
+
+head(Carseats)
+
+lm.fit <- lm(Sales ~ . + Income:Advertising + Price:Age, Carseats)
+summary(lm.fit)
+
+attach(Carseats)
+contrasts(ShelveLoc)
+
+?contrasts
+
+# Exercise 6.1
+
+babyweight <- read.table("babyweights.txt", header = TRUE)
+lm.fit <- lm(bwt ~ smoke, babyweight)
+summary(lm.fit)
+
+#>(a) Write the equation of the regression line
+#>      bwt.approx = 123.05 - 8.94(smoke)
+#>  
+#>(b) Interpret the slope in this context, and calculate the predicted birth weight of babies
+#>    born to smoker and non-smoker mothers.
+#>      The intercept over here is the average birth weight (in ounces) predicted by the model
+#>      for a baby whose mother did not smoke at all during pregnancy.
+#>      
+#>      Predicted non-smoke = 123.05
+#>      Predicted smoke = 123.05 - 8.94 = 114.11
+#>      
+#>(c) Is there a statistically significant relationship between the average bwt and smoking?
+#>      The p-value here is much less than 0 (listed here as 0.0000) so it is very small so there
+#>      is a significant relationship between avg bwt and smoking. 
+
+# Exercise 6.2
+
+lm.fit <- lm(bwt ~ parity, babyweight)
+summary(lm.fit)
+
+#>(a) Write the equation of the regression line.
+#>      120.07 - 1.93(parity)
+#>      
+#>(b) Interpret the slope in this context, and calculate the predicted birth weight of first borns and others.
+#>      The slope here is the predicted average bwt of a first born baby.
+#>      
+#>      Predicted first born = 120.07
+#>      Predicted non-first born = 120.07 - 1.93 = 118.14
+#>
+#>(c) Is there a statistically significant relationship between avg bwt and parity?
+#>      The p-value here is 0.1052 so there is not a significant relationship between avg bwt and parity.
+
+# Exercise 6.3
+
+lm.fit <- lm(bwt ~ ., babyweight)
+summary(lm.fit)
+
+#>(a) Write the equation of the regression line that includes all of the variables.
+#>      -80.41 + 0.44(gestation) - 3.33(parity) - 0.01(age) + 1.15(height) + 0.05(weight) - 8.40(smoke)
+#>      
+#>(b) Interpret the slopes of gestation and age in this context.
+#>      The slope for gestation here indicates a predicted increase of 0.44 ounces for every
+#>      day of gestation. The slope for age shows a predicted decrease of 0.01 ounces for every 
+#>      year of the mother's age. 
+#>      
+#>(c) The coefficient for parity is different than in the linear model in exercise 6.2. 
+#>    Why might there be a difference?
+#>      This might be bc the pattern of effect is different when comparing just bwt against parity
+#>      vs. in addition to all the other variables. I.e. The relationship between parity and bwt is different
+#>      than the relationship between bwt and parity in addition with the other variables in the dataset. 

```

Comment:
This is a pretty good explanation, but it would be nice to also get into why it's different, and what it is about the new variables that could be causing this change.